phpstan-baseline.neon

```diff
@@ -0,0 +1,27 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Class Sabberworm\\\\CSS\\\\Parsing\\\\ParserState constructor invoked with 1 parameter, 2\\-3 required\\.$#"
+			count: 1
+			path: lib/Sabberworm/CSS/CSSList/CSSList.php
+
+		-
+			message: "#^Access to an undefined property Sabberworm\\\\CSS\\\\Parsing\\\\ParserState\\:\\:\\$oParserHelper\\.$#"
+			count: 1
+			path: lib/Sabberworm/CSS/Parsing/ParserState.php
+
+		-
+			message: "#^Access to an undefined property Sabberworm\\\\CSS\\\\Property\\\\Charset\\:\\:\\$aComments\\.$#"
+			count: 5
+			path: lib/Sabberworm/CSS/Property/Charset.php
+
+		-
+			message: "#^Class PHPUnit\\\\Framework\\\\TestCase referenced with incorrect case\\: PHPunit\\\\Framework\\\\TestCase\\.$#"
+			count: 1
+			path: tests/Sabberworm/CSS/OutputFormatTest.php
+
+		-
+			message: "#^Class PHPUnit\\\\Framework\\\\TestCase referenced with incorrect case\\: PHPunit\\\\Framework\\\\TestCase\\.$#"
+			count: 1
+			path: tests/Sabberworm/CSS/ParserTest.php
+
```

Comment:
Cool, I thought there’d be more

Reply:
That's also because we're starting at level 0. (It goes up to level 8 …)